hr_timesheet_predefined_description/views/hr_timesheet_views.xml

```diff
@@ -0,0 +1,40 @@
+<odoo>
+    <record id="view_analytic_line_form_inherit" model="ir.ui.view">
+        <field name="name">account.analytic.line.form.inherit</field>
+        <field name="model">account.analytic.line</field>
+        <field name="inherit_id" ref="hr_timesheet.hr_timesheet_line_tree" />
+        <field name="arch" type="xml">
+            <field name="name" position="attributes">
```

Comment:
Don't hide the name field. Make it readonly (with `force_save="1"`) if the predefined description is set.